docs/topics_5/plugin-resolve-issues.adoc

```diff
@@ -1,24 +1,24 @@
 // Module included in the following assemblies:
 // * docs/plugin-guide_5/master.adoc
 [id='resolve_issues_{context}']
-= Resolving {ProductShortName} Issues
+= Resolving {ProductShortName} issues
 
 You can resolve {ProductShortName} issues by updating the code manually or by applying a quick fix when available.
 
-== Resolving an Issue Manually
+== Resolving an issue manually
 
 Review the {ProductShortName} issue details and additional resources and update the source code as necessary. When you update a line of code marked as an {ProductShortName} issue, the {ProductShortName} issue will be marked with the stale icon (image:stale_issue.gif[Stale issue icon]) until the next time that {ProductShortName} is run on the project.
```

Comment:
Remove extra space at beginning of lines 10 and 12.

Reply:
They just look spaces.